test/e2e/common/container_probe.go

```diff
@@ -413,10 +416,14 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		startedTime, err := GetContainerStartedTime(p, "busybox")
 		framework.ExpectNoError(err)
 
+		startedIn := readyTime.Sub(startedTime)
 		framework.Logf("Container started at %v, pod became ready at %v", startedTime, readyTime)
-		if readyTime.Sub(startedTime) < 40*time.Second {
+		if startedIn < sleepBeforeStarted*time.Second {
 			framework.Failf("Pod became ready before startupProbe succeeded")
 		}
+		if startedIn > (sleepBeforeStarted+5)*time.Second {
+			framework.Failf("Pod became ready in %v, more than 5s after startupProbe succeeded", startedIn)
```

Comment:
`5s` is lower than `10s` default interval for the readiness probe, this is why it's 5, correct? This test case will benefit from more extended comment on what exactly it tests. Maybe:

 ```suggestion
			framework.Failf("Pod became ready in %v, more than 5s after startupProbe succeeded. It means that the delay readiness probes were not initiated immediately after startup finished.", startedIn)
```